packages/playwright/src/generators/visualRegression/utils/update-tasks-yamls.ts

```diff
@@ -27,6 +27,12 @@ export function updateTasksYaml(
                           'npx nx affected --base="$BASE_SHA" --target=e2e --parallel=1',
                       ],
                   },
+                  'e2e:updatesnapshots': {
+                      description: 'Run e2e snapshot updates in ci',
+                      command: [
+                          'npx nx affected --base="$BASE_SHA" --target=e2e --parallel=1 --skip-nx-cache --update-snapshots --grep @visual-regression',
+                      ],
+                  },
```

Comment:
Just confirming, can't quite tell, but is this only added if visual regression == native?

Reply:
Only called when native